src/trace_writer.ml

```diff
@@ -9,7 +9,7 @@ let is_kernel_address addr = Int64.(addr < 0L)
    compensate, the trace writer subtracts the time of the first event from all time spans, producing
    what we call a "mapped time". Only mapped times may be written to the trace file. *)
 module Mapped_time : sig
-  type t = private Time_ns.Span.t [@@deriving sexp_of, compare]
+  type t = Time_ns.Span.t [@@deriving sexp_of, compare]
```

Comment:
We want to keep this type private so we don't accidentally pass a regular `Time_ns.Span.t` to a function that requires `Mapped_time.t`. If you have a place where this is definitely necessary, you can use the `:>` operator.